services/blockchain-indexer/shared/dataService/transactions.js

```diff
@@ -132,6 +132,11 @@ const dryRunTransactions = async params => {
 	return response;
 };
 
+const estimateTransactionFees = async params => {
+	const response = await dataService.estimateTransactionFees(params);
+	return response;
+};
```

Comment:
should we reassign data and meta for the sake of consistency? Ex:
https://github.com/LiskHQ/lisk-service/pull/1629/files#diff-ade3c47f112f949d23dca23d591644e8264916b752b7766a7fcf0d1ffeffec07R91-R104